docs/platform/howto/byoc/create-custom-cloud.md

```diff
@@ -112,8 +112,16 @@ you informed on the progress.
 -   You have the [Aiven CLI client](/docs/tools/cli) installed.
 -   You have the [super admin](/docs/platform/howto/make-super-admin) role in your Aiven
     organization.
--   You have Terraform installed.
--   You have required [IAM permissions](#iam-permissions)
+-   You have [Terraform](/docs/tools/terraform) installed.
+-   You have required [IAM permissions](#iam-permissions).
+-   You have extracted the ID of your Aiven organization to be connected
+    with your own cloud account to create the custom cloud:
+
+    - From the output of the `avn organization list` command
+    - By going to <ConsoleLabel name="organizations"/> in the
+      [Aiven Console](https://console.aiven.io/) (either from the organizations selector
```

Comment:
![Screenshot 2024-09-05 at 10 15 31](https://github.com/user-attachments/assets/3712e502-819f-4ce9-bdd2-2f99557f5dd4)

Reply:
Organization selector in the console:

![Screenshot 2024-09-05 at 10 17 38](https://github.com/user-attachments/assets/cc6682e9-b175-4748-b663-26a905f45998)


By clicking the selector > Manage organization you get the Organizations page:

![Screenshot 2024-09-05 at 10 18 50](https://github.com/user-attachments/assets/07b10907-b18c-40d3-9f9a-9311563804d9)

Reply:
Ah, okay, so it should be org selector > Manage organizations > Organizations. But, it might be better to only show one path to make this easier to follow. In this case, I'd go with the User info > Organizations because it's faster and because when you use the org selector you need to go up to the org level by clicking the arrow to get the "Manage organizations" option, which takes a lot of explaining.

Reply:
sure, updated :-) Thanks!